config.json

```diff
@@ -46,6 +46,17 @@
         "Transforming"
       ]
     },
+	{
+      "uuid": "2ee3cc7a-db3f-4668-9983-ed6d0fea95d1",
+      "slug": "two-fer",
+      "core": true,
```

Comment:
@robkeim Well actually, I have to update the C# track as it doesn't unlock any exercises there ATM. The way I decided which exercises to unlock, is to check which 5/6 exercises have one or more matching topics and were not unlocked by a previous core exercise.